cmds/webboot/utils.go

```diff
@@ -114,3 +114,23 @@ func validURL(url string) (string, string, bool) {
 		return url, "Invalid URL.", false
 	}
 }
+
+func fileExists(filepath string) bool {
+	info, err := os.Stat(filepath)
+	if err != nil {
+		return false
+	} else {
+		return !info.IsDir()
```

Comment:
This is dangerous. I think you should do a full check for it being a regular file.

Also, when you have a return as on line 121, the convention is that you don't use an else.

since it is a bool you might even do
return err != nil && info.IsRegular()
on line 120

Reply:
That makes sense, I updated the function to this:

```go
func fileExists(filepath string) bool {
	info, err := os.Stat(filepath)
	return err == nil && info.Mode().IsRegular()
}
```